From 252af01367828401e01cfd90fb05167f30b994f0 Mon Sep 17 00:00:00 2001
From: Deepak Rathore <deepak.rathore@einfochips.com>
Date: Fri, 24 Mar 2023 16:20:17 +0530
Subject: [PATCH 5/5] onnxruntime-native: Fix compilation issue of onnxruntime

- onnxruntime-native package is failed to compile as it is unable
  to fetch source so fixed compilation issue.

Signed-off-by: Deepak Rathore <deepak.rathore@einfochips.com>

diff --git a/meta-ml/recipes-libraries/onnxruntime/onnxruntime-native_1.8.1.bb b/meta-ml/recipes-libraries/onnxruntime/onnxruntime-native_1.8.1.bb
index d2c22c3b7..57a4a2377 100644
--- a/meta-ml/recipes-libraries/onnxruntime/onnxruntime-native_1.8.1.bb
+++ b/meta-ml/recipes-libraries/onnxruntime/onnxruntime-native_1.8.1.bb
@@ -6,7 +6,7 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=37b5762e07f0af8c74ce80a8bda4266b"
 
 # For ONNX Runtime v0.3.0 we need only 'protoc' to be compiled for native system
 # 3.16.0
-GIT_protobuf = "git://github.com/google/protobuf.git;branch=master;name=protobuf"
+GIT_protobuf = "git://github.com/google/protobuf.git;branch=main;name=protobuf;protocol=https"
 SRCREV_protobuf = "2dc747c574b68a808ea4699d26942c8132fe2b09"
 
 SRC_URI = "\
-- 
2.17.1

